src/utils/initiative-utils.ts

```diff
@@ -203,18 +203,54 @@ export class InitiativeBuilder {
 		}
 		turnText += '\n';
 		if (actorsInGroup.length === 1 && actorsInGroup[0].name === actorGroup.name) {
-			//We're just displaying the actor in its initiative slot
+			//We're displaying the actor in its initiative slot with HP, Max HP, SP and Max SP shown if present
 			turnText += `${extraSymbol} ${actorGroup.initiativeResult}: ${actorGroup.name}`;
+			turnText += this.pointsToText(actorsInGroup[0]);
 		} else {
 			turnText += `${extraSymbol} ${actorGroup.initiativeResult}: ${actorGroup.name}\n`;
 			const sortedActors = actorsInGroup.sort((a, b) => a.name.localeCompare(b.name));
 			for (let i = 0; i < sortedActors.length; i++) {
-				turnText += `       ${i}. ${sortedActors[i].name}\n`;
+				turnText += `       ${i}. ${sortedActors[i].name}`;
+				turnText += this.pointsToText(sortedActors[i]);
+				turnText += `\n`;
 			}
 		}
 		return turnText;
 	}
 
+	private pointsToText(actor: InitiativeActor): string {
```

Comment:
I like having very clear names for my functions. This is a little vague. Can we call it something like `generateActorStatDisplayString`. It's verbose, but I want the functionality of functions to be understandable at a glance without looking at the implementation. Another thing you could do is lump the actor name in here as well, and have it generate the whole string for something like  `generateActorTurnString`

Reply:
I do agree that this name was vague, I'd meant to go back and come up with a better name at the time of writing it and spaced. I'm hesitant to include the actor name as with that moved into this function it is less clear looking what the formatting is attached to in the preceding lines at a glance. If I went back to change the formatting for what was right before the name, I think it would be easier with the name not factored into the function.